### ISSUE TITLE
convert softmax

### ISSUE THREAD

**Issue by ShiraLightricks** [onnx/keras-onnx#511]
Hi,
I have a problem with the softmax activation layer, it seems the code is supposed to handle this case but it gives me the following error:

Unsupported activation in the layer(<function softmax at 0x1365b6320>)

details:
python=3.7, keras2onnx=1.7, tensorflow=2.2.0, onnx=1.5

Thanks!
Shira

**Comment by jiafatom** [comment]
Please give more information about the softmax layer, is it a tf layer? Since we don't have the model, can you give the stack trace?

**Comment by shonigs** [comment]
It seems that this doesn't work  
`model.layers[-1].activation = tf.keras.activations.softmax`
and this does 
```
model.layers[-1].activation = tf.keras.activations.linear
output = tf.keras.layers.Softmax()(model.output)
model=tf.keras.Model(inputs=model.inputs, outputs=output)`
```

**Comment by jiafatom** [comment]
Fixed [here](https://github.com/onnx/keras-onnx/pull/513), please pull keras2onnx master and try again.
